Storage/src/Connection/Rest.php

```diff
@@ -100,6 +100,26 @@ class Rest implements ConnectionInterface
      */
     private $restRetryFunction;
 
+    /**
+     * @var string
+     */
+    private $retryStrategy;
```

Comment:
add typehint for string! We can't typehint `callable`, but we can typehint `string` so we should!
```suggestion
    /**
     * @var string|null
     */
    private ?string $retryStrategy;
```